src/components/AccountDropdownComponent.jsx

```diff
@@ -312,10 +315,23 @@ class AccountDropdownComponent extends Component<Props, State> {
               })}
             {groupedCryptoAccounts.length !== 0 && <Divider />}
             <MenuItem value='addCryptoAccounts'>
```

Comment:
clicking the menu item without touching btn causing dropdown to be closed. is it possible to make disabled for "send"?

Reply:
Disable this MenuItem causes all children components to be disabled.